frontend/src/components/CardBack.tsx

```diff
@@ -232,12 +243,17 @@ const ExplanationContent = ({
 const ExamplesContent = ({
   examples,
   textSecondaryColor,
+  headerColor,
+  accentColor,
 }: ExamplesContentProps) => {
   const [showAllExamples, setShowAllExamples] = useState(false);
   const [showButton, setShowButton] = useState(false);
 
   const { theme, colors, computed } = useTheme();
 
+  const resolvedHeaderColor = headerColor ?? computed.headerColor;
+  const resolvedAccent = accentColor ?? computed.accent;
```

Comment:
This doesn't make a lot of sense imo. We should just use the the theme colors, why variable props here?